tests/integration/conftest.py

```diff
@@ -1,7 +1,9 @@
-from helpers.cluster import run_and_check
+#!/usr/bin/env python3
+
 import pytest
 import logging
 import os
+from helpers.cluster import run_and_check
```

Comment:
very minor thing, but the empty line between stdlib and local imports improves readability